_includes/ballot-nav.html

```diff
@@ -11,7 +11,17 @@
           <h4>{% include referendum_measure_heading.html referendum_measure_display=locality.referendum_measure_display %}</h4>
           {% for ballot_item in referendums %}
               {% assign referendum = site.referendums | where: "path", ballot_item | first %}
-              <p><a href="{{ referendum.url | prepend: site.baseurl }}">{% if referendum.number %}{{ referendum.number }} {% endif %}{{ referendum.title | escape}}</a></p>
+
+               {% if page.url == referendum.url %}
+               <p class="current-page">
+                 <a href="{{ referendum.url | prepend: site.baseurl }}">{% if referendum.number %}{{ referendum.number }} {% endif %}{{ referendum.title | escape}}</a>
+               </p>
+               {% else %}
+               <p>
+                 <a href="{{ referendum.url | prepend: site.baseurl }}">{% if referendum.number %}{{ referendum.number }} {% endif %}{{ referendum.title | escape}}</a>
+               </p>
+               {% endif %}
```

Comment:
This `if` statement is a little broad, you could just surround the class attribute instead of around the whole element. e.g.

```
		<p {% if page.url == referendum.url %}class="current-page"{% endif %}>
                  <a href="{{ referendum.url | prepend: site.baseurl }}">{% if referendum.number %}{{ referendum.number }} {% endif %}{{ referendum.title | escape}}</a>
                </p>
```

Reply:
Got it, thanks @adborden !